### PR TITLE
ADX-285-versioning-with-react

### PR DESCRIPTION
**Pull request by Manoj-nathwani**
Made a PR as I want to double check this is ok:
https://github.com/fjelltopp/adx_develop/pull/55/files#diff-d397ae6f0ef094a653e50c596eac39cb61b271c5330852436599c2c9f561783aR124

### CODE REVIEW

File: ckan/adx_config.ini [hunk 1]
```diff
@@ -121,15 +121,15 @@ ckan.redis.url = redis://redis:6379/1
 #		Add ``resource_proxy`` to enable resorce proxying and get around the
 #		same origin policy
 # Note: Plugins to the left take precendence over plugins to the right!!! Opposite to what you might expect.
-ckan.plugins = external_storage unaids  emailasusername restricted authz_service versions stats text_view image_view
+ckan.plugins = scheming_datasets external_storage unaids emailasusername restricted authz_service versions stats text_view image_view
```
Comment (Manoj-nathwani) [inline ckan/adx_config.ini:124]: @tomeksabala This is important to double check won't cause any unexpected issues

File: util/ckan_loader.py [hunk 2]
```diff
@@ -150,4 +150,4 @@ def load_data(adr_url, apikey):
 
 
 if __name__ == '__main__':
-    load_data('http://ckan:5000', "6011357f-a7f8-4367-a47d-8c2ab8059520")
+    load_data('http://ckan', "6011357f-a7f8-4367-a47d-8c2ab8059520")
```
Comment (tomeksabala) [inline util/ckan_loader.py]: This is only a hint on how to use the module. You could even consider it a piece of doc.
If we want to change it it should say `http://dev-adr` IMO.

File: ckan/adx_config.ini (hunk 1 above)
Comment (tomeksabala) [inline ckan/adx_config.ini]: why do you need to change the order of the `scheming ` plugin?
Also could you fix the formatting of this config value? 1st line if super long while the last two are much shorter then the others.

**Review by tomeksabala — CHANGES_REQUESTED** [review]

File: ckan/adx_config.ini (hunk 1 above)
Comment (Manoj-nathwani) [reply ckan/adx_config.ini]: This was the only way unaids could override the scheming `upload.html` template here: https://github.com/fjelltopp/ckanext-unaids/pull/105/files#diff-8e5ca759def387b362229130f1827669ae72fe441c9884c2c4a59a617b3fd295

My override wasn't working until I moved scheming to before unaids in `ckan.plugins`

Comment (Manoj-nathwani) [reply ckan/adx_config.ini]: @tomeksabala - After our call this morning, can you confirm this is ok now?

Comment (tomeksabala) [reply ckan/adx_config.ini]: Only this last bit: >Also could you fix the formatting of this config value? 1st line if super long while the last two are much shorter then the others.

Comment (Manoj-nathwani) [reply ckan/adx_config.ini]: Done

**Comment by tomeksabala** [comment]
I think it looks good now. One more thing should be done (I'm working on this atm): move `ckanext-blob-storage` from extensions installed from local sources to directly installing from github datopian fork.

**Review by tomeksabala — APPROVED** [review]
Look good to me.

**Pull request merged**